src/.eslintrc.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
```

Comment:
why is this file in src and not in root folder ?
are you sure that main eslint config has access to this file ?